### ISSUE TITLE
Support translations, default to device language

### ISSUE THREAD

**Issue by score76** [jfoclpf/form-for-parking-violation#155]
Down here in the Algarve there are some municipalities where for up to 26% of residents Portuguese is not the first language. The app is being written about in The Portugal News, an English language newspaper, this will generate interest from lots of expats. Parking is also awful down here, particularly when the tourists arrive for the summer.

**Comment by jfoclpf** [comment]
Thank you very much for your comment and I'm really glad the app is being spoken in Portugal News. To be honest I never though about translating it to English, but now that you mentioned it, I might consider it. Don't forget though that the final message to authorities must be written in Portuguese, but indeed the User Interface might be in English as well.

**Comment by score76** [comment]
Yes, exactly what I meant, have the UI translatable but the output email in Portuguese. For your interest, the news story is here:
https://www.theportugalnews.com/news/2023-05-24/new-app-to-report-illegally-parked-cars/77914
Happy to help provide the translation if you need help.

**Comment by jfoclpf** [comment]
We should use this solution once I answer in stackoverflow
https://stackoverflow.com/a/47612798/1243247

@score76 would you like to help in the translation to English? Should not be hard

**Comment by score76** [comment]
Yes sure. Happy to help with that 👍🏻
